middleware/src/main/java/com/frontegg/sdk/middleware/identity/impl/IdentityService.java

```diff
@@ -0,0 +1,108 @@
+package com.frontegg.sdk.middleware.identity.impl;
+
+import com.auth0.jwt.JWT;
+import com.auth0.jwt.JWTVerifier;
+import com.auth0.jwt.algorithms.Algorithm;
+import com.auth0.jwt.interfaces.Claim;
+import com.auth0.jwt.interfaces.DecodedJWT;
+import com.frontegg.sdk.api.client.IApiClient;
+import com.frontegg.sdk.common.exception.FronteggSDKException;
+import com.frontegg.sdk.common.util.HttpUtil;
+import com.frontegg.sdk.config.FronteggConfig;
+import com.frontegg.sdk.middleware.authenticator.FronteggAuthenticator;
+import com.frontegg.sdk.middleware.context.FronteggContext;
+import com.frontegg.sdk.middleware.context.FronteggContextHolder;
+import com.frontegg.sdk.middleware.identity.IIdentityService;
+import com.frontegg.sdk.middleware.identity.model.IdentityModel;
+import org.bouncycastle.util.encoders.Base64;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import sun.security.rsa.RSAPublicKeyImpl;
```

Comment:
I think you are not supposed to use sun's apis anymore. Can you find a replacement with spring security?